README.md

```diff
@@ -1,177 +1,164 @@
+1. Now that your operator is set up, you need to create persistent volume in the name of `fp-predict-plus-pv` to be bound to persistent volume claims. This step is necessary to enable storage capabilities for the operator to manage datasets. In the left navigation, Click **Storage > Persistent volume**. Update the name to `fp-predict-plus-pv` and set the storage capacity to 20 GB and hit **Create**. If you need more storage, you can increase it in the YAML file and create the persistent volume accordingly.
 
-![](https://github.com/IBM/getting-started-with-fppredictplus/blob/master/images/create-pv.png)
+    ![](https://github.com/IBM/getting-started-with-fppredictplus/blob/master/images/create-pv.png)
 
-We need to create a persistent volume claim (storage) to use the storage created in previous step and bind it to the instance of this operator. Click on storage under Web Console and select Persistent Volume Claims.
+1. You need to create a persistent volume claim (storage) to use the storage created in the previous step and bind it to the instance of this operator. On the web console, click on **Storage** and select **Persistent Volume Claims**.
 
-![](https://github.com/IBM/getting-started-with-fppredictplus/blob/master/images/select-pvc.png)
+    ![](https://github.com/IBM/getting-started-with-fppredictplus/blob/master/images/select-pvc.png)
 
-The next step is to create persistent volume claim. We can select Storage Class from Gold, Silver, Bronze which was created in previous step, give the name as `fp-predict-plus-pvc`, select single user access & assign the storage size as 20 GB. 
+1. The next step is to create a persistent volume claim. Choose the storage class you need &mdash; either gold, silver, or bronze. Add your name as `fp-predict-plus-pvc`, select single user access, and assign the storage size as 20 GB. 
 
-![](https://github.com/IBM/getting-started-with-fppredictplus/blob/master/images/create-pvc.png)
+    ![](https://github.com/IBM/getting-started-with-fppredictplus/blob/master/images/create-pvc.png)
 
-After the persistent volume claim (PVC) is created, it needs to be bound with the persistent volume (PV) which was created in earlier step. We should see the status as `Bound` per below after couple of minutes.
+1. After you create the persistent volume claim (PVC), you need to bind it with the persistent volume (PV) which you created in tye earlier step. You should see the status as `Bound` per below after couple of minutes.
 
-![](https://github.com/IBM/getting-started-with-fppredictplus/blob/master/images/pvc-bound.png)
+    ![](https://github.com/IBM/getting-started-with-fppredictplus/blob/master/images/pvc-bound.png)
 
-### Install the operand (Instance) of FPPredict Plus
+## Install the instance of FPPredict Plus
 
```

Comment:
It is actually called operand however some people might prefer it as Instance. Hence both should be retained.